app-drawer-layout/app-drawer-layout.html

```diff
@@ -233,16 +233,22 @@
             contentContainer.style.marginRight = '';
           }
         }
-        if (drawer) {
+        if (drawer && this._drawerNeedsReset) {
```

Comment:
@keanulee did you modify iron-resizable-behavior, so that `iron-request-resize-notifications` fires from a raf?